package.json

```diff
@@ -12,33 +12,26 @@
     "@babel/core": "^7.22.5",
     "@babel/preset-env": "^7.22.5",
     "@babel/preset-typescript": "^7.22.5",
+    "@biomejs/biome": "^1.7.3",
     "@jest/globals": "^29.4.3",
     "@rollup/plugin-typescript": "^8.2.0",
     "@swc/core": "^1.3.66",
     "@swc/jest": "^0.2.26",
     "@types/benchmark": "^2.1.5",
     "@types/jest": "^29.2.2",
     "@types/node": "^20.0.0",
-    "@typescript-eslint/eslint-plugin": "^7.8.0",
-    "@typescript-eslint/parser": "^7.8.0",
     "@web-std/file": "^3.0.3",
     "babel-jest": "^29.5.0",
     "benchmark": "^2.1.4",
     "console-table-printer": "^2.12.0",
     "dependency-cruiser": "^9.19.0",
-    "eslint": "^9.2.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-ban": "^1.6.0",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-simple-import-sort": "^7.0.0",
-    "eslint-plugin-unused-imports": "^2.0.0",
+    "execa": "^9.1.0",
     "husky": "^7.0.4",
     "jest": "^29.3.1",
     "lint-staged": "^12.3.7",
     "mitata": "^0.1.11",
     "nodemon": "^2.0.15",
     "prettier": "^2.6.0",
```

Comment:
Can remove this dependency too. Prettier seems to be unused.

Reply:
> Can remove this dependency too. Prettier seems to be unused.

Prettier still required for markdown formatting